repos/fdbt-site/src/pages/products/services.tsx

```diff
@@ -126,15 +132,27 @@ export const matchProductsToServices = (
         return map;
     }, new Map<string, MyFaresProduct[]>());
 
-    return services.map((service) => ({
-        ...service,
-        products:
-            productsByLine
-                .get(service.lineId)
-                ?.filter((product) =>
-                    showProductAgainstService(product.startDate, product.endDate, service.startDate, service.endDate),
-                ).length ?? 0,
-    }));
+    return services.map((service) => {
+        const serviceWithEndDatesSwapped = {
```

Comment:
Could we avoid defining this variable, since it's only used once, and instead override endDate in the return statement?

Reply:
@matt-twomey i did consider this, but was unsure how to write it. Do you have an example?